wp-content/themes/pawar2018/header.php

```diff
@@ -37,8 +37,10 @@
             <img src="<?php echo get_bloginfo('template_url') ?>/assets/logo.svg" alt="Ameya Pawar 2018 Logo">
         </a>
 
-		<div class="header-burger">
-		  <div class="header-burger__icon"></div>
+		<div class="header-burger__icon">
```

Comment:
Losing the `header-burger` class results in the 🍔 menu being visible at all breakpoints. I think you just need to apply the `.header-burger` `64rem` breakpoint to the `.header-burger__icon` selector. (And then maybe get ride of the `.header-burger` selector?)